.pre-commit-config.yaml

```diff
@@ -23,8 +23,9 @@ repos:
   hooks:
     - id: black
       language_version: python3
-# - repo: https://github.com/pre-commit/mirrors-clang-format
-#   rev: v18.1.5
-#   hooks:
-#     - id: clang-format
-#       types_or: [c++, c]
+- repo: https://github.com/pre-commit/mirrors-clang-format
+  rev: v18.1.5
+  hooks:
+    - id: clang-format
+      types_or: [c++, c]
+      files: ^ttnn/.*\.(hpp|cpp)$
```

Comment:
You don't need the `.*\.(hpp|cpp)$` file type filter here, `^ttnn/` should work. The `types_or` would select the c/c++ files already (unless there are other extensions in the filter you wanted to avoid). Extensions selected are here if you search for c/c++ https://github.com/pre-commit/identify/blob/main/identify/extensions.py

Reply:
Thanks, updated